src/constants.ts

```diff
@@ -89,9 +94,33 @@ export const kChildCollectionAttributes = [
     title: "Season",
     type: "categorical",
     hasToken: true
+  },
+  {
+    name: "sunriseMinSinceMidnight",
+    title: "Sunrise Minutes Since Midnight",
+    type: "numeric",
+    hasToken: true
+  },
+  {
+    name: "sunsetMinSinceMidnight",
+    title: "Sunset Minutes Since Midnight",
+    type: "numeric",
+    hasToken: true
   }
 ];
 
 export const kDefaultOnAttributes = [
-  "date", "sunrise", "sunset", "dayLength"
+  "date", "sunrise", "sunset", "dayLength", "season", "sunlightAngle", "solarIntensity", "sunriseMinSinceMidnight", "sunsetMinSinceMidnight", "dayNumber"
 ];
+
+export const kDateWithTimeFormats = {
+  asZuluISO: "YYYY-MM-DDTHH:mm[Z]",              // 1999-01-23T21:45Z
+  asLocalISOWithTZOffset: "YYYY-MM-DDTHH:mmZ",   // 1999-01-23T14:45-07:00
+  asClockTimeString: "HH:mm",                    // 14:45
+  asClockTimeStringAMPM: "h:mm a",               // 2:45 PM
+}
+
+export const kDateFormats = {
```

Comment:
If we end up using formulas for the date display column we will not need these, these are mostly here for making iteration easier.